routes/categories.js

```diff
@@ -3,46 +3,48 @@ var router = express.Router();
 var db = require('../db.js');
 
 router.route('/')
-  .get(function(req, res) {
```

Comment:
There needs to be a GET route to get all the categories for the given user. Right now, this isn't possible because the caller's user id will be stored in a session, which we don't have yet. You need to keep the `GET /categories` in place though. Leave the static data as-is

Reply:
This means you will also need to create a db query that gets all books `WHERE user_id = ?` but that can come later on after we have sessions going